### PR TITLE
feat(loyalties): Add schemas for systems, accounts, cards

### PR DESCRIPTION
**Pull request by qtotuan**

### CODE REVIEW

File: lib/v0/loyalty_accounts/base.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  customer: {
+    description: 'The customer that this account belongs to.',
+    type: 'string',
+    format: 'uuid',
+    example: '936835f7-2d75-41d2-9001-38ed6e458328'
+  },
+  name: oneOf({
+    type: 'string',
+    example: 'Tillhub Loyalty Account',
+    minLength: 1,
```
Comment (Endebert) [inline lib/v0/loyalty_accounts/base.js]: Any specific reason we allow `null`, but not an empty string?

File: lib/v0/loyalty_cards/base.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+module.exports = {
+  code: {
+    type: 'string'
+  },
+  activated_at: oneOf({
+    type: 'string',
+    format: 'date-time'
+  }),
+  name: oneOf({
+    type: 'string'
```
Comment (Endebert) [inline lib/v0/loyalty_cards/base.js]: You try to exclude empty string for most other properties, why not for this one as well?

File: lib/v0/loyalty_logs/create.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const base = require('./base')
+const defaultResponse = require('./default-response')
+
+module.exports.request = {
+  $id: 'https://schemas.tillhub.com/v0/loyalty_logs.create.request.schema.json',
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  additionalProperties: false,
+  type: 'object',
+  required: [],
```
Comment (Endebert) [inline lib/v0/loyalty_logs/create.js]: Just to double-check: so sending a completely empty create request is still valid? What happens in this case?

Comment (qtotuan) [reply lib/v0/loyalty_logs/create.js]: Ah, that's a mistake, there is at least one required field. I'll add it

File: lib/v0/loyalty_cards/base.js (hunk 2 above)
Comment (qtotuan) [reply lib/v0/loyalty_cards/base.js]: You're right, I'll add it here as well.

**Comment by qtotuan** [comment]
I'm gonna merge it, because it was just a small fix

File: lib/v0/loyalty_logs/create.js (hunk 3 above)
Comment (Endebert) [reply lib/v0/loyalty_logs/create.js]: :ok_hand:

**Pull request merged**